AI-and-Analytics/Getting-Started-Samples/INC-Quantization-Sample-for-PyTorch/README.MD

```diff
@@ -64,6 +54,7 @@ The sample contains one Jupyter Notebook and one Python script.
 
 ## Run the `Getting Started with Intel® Neural Compressor for Quantization` Sample
 
+### On Linux* (Only applicable to Intel AI Tools Offline Installer)
```

Comment:
Change heading to "Steps for Intel AI Tools Offline Installation"

Reply:
addressed it accordingly.